common/persistence/sql/sqlplugin/mysql/execution_maps.go

```diff
@@ -129,44 +129,7 @@ var (
 
 // ReplaceIntoActivityInfoMaps replaces one or more rows in activity_info_maps table
 func (mdb *db) ReplaceIntoActivityInfoMaps(rows []sqlplugin.ActivityInfoMapsRow) (sql.Result, error) {
-	q, args, err := mdb.db.BindNamed(setKeyInActivityInfoMapQry, rows)
-	if err != nil {
-		return nil, err
-	}
-
-	q = expandBatchInsertQuery(q, len(rows))
-
-	return mdb.conn.Exec(q, args...)
-}
-
-func expandBatchInsertQuery(q string, rowCount int) string {
```

Comment:
So is this all fixed inside `sqlx` itself now?

Reply:
yep